docs/plot_result.rst

```diff
@@ -0,0 +1,34 @@
+####################
+Plotting fit results
+####################
+
+Plotting
+========
+
+The saved results from a PAHFIT run can be plotted using the ``plot_pahfit``
+script.  By default, the plot is displayed.  To save the plot instead,
+use the "--savefig" command line option.
+
+.. code-block:: console
+
+  $ plot_pahfit M101_Nucleus_irs.ipac M101_Nucleus_irs_output.ipac
+
+The first argument gives the file with the observed spectrum.
+The second argument gives the file with the model pack file to be used.
+
+Help for the possible commandline options for the ``plot_pahfit`` script
+can be seen by:
+
+.. code-block:: console
+
+  $ plot_pahfit --help
+
+Example Plot
+============
+
+The fit to the M101 Nucleus spectrum gives the observed spectrum as
+open squares, the full PAHFIT model as a green line, the dust continua
+as red lines, the dust features as blue lines, the atomic features
+as purple lines, and the attenuation as a dashed, black line.
```

Comment:
Could it be the sum of the continua is also plotted in some dark purple?
Should we give an example with also star-light continuum?

Reply:
Classic PAHFIT has the sum of all continuum components as a thick gray line. RGB=[100,100,100].

Reply:
The continuum is plotted as a black line.  I've added text to the docs to say this.